packages/client/base/src/types/embed/crypto.ts

```diff
@@ -47,6 +47,7 @@ type SolanaTransaction = _SolanaTransaction;
 export type ETHEmbeddedCheckoutSigner = CommonEmbeddedCheckoutSignerProps & {
     signAndSendTransaction: (transaction: EthersTransaction) => Promise<string>;
     network?: Blockchain;
+    supportedNetworks?: Blockchain[];
```

Comment:
i think you need to update `updatableParams` file also

Reply:
Mmmmm I think this is not supposed to change. Otherwise we open a bunch of other scenarios, like what happens if they change supportedNetworks and the selected network is not on that list anymore. So it was intentionally made "not updatable". thoughts?

Reply:
oops i meant whatever place the params get serialized into the url, im pretty sure the signer has some custom logic, so not sure if this would get passed through or not

Reply:
ah! no, the signer is transformed into a JSON.stringify so it gets it directly as it is 🙏